src/frontend/app/features/applications/cli-info-application/cli-info-application.component.html

```diff
@@ -1,12 +1,5 @@
 <app-page-header [breadcrumbs]="breadcrumbs$ | async">
   <h1>CLI Info</h1>
-  <div class="page-header-right">
```

Comment:
There are other places where we have the close button, so we should be consistent and remove them there too.